style.css

```diff
@@ -0,0 +1,586 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,300;1,400;1,700&family=Poppins:wght@400;700&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Lato", sans-serif;
+  font-style: normal;
+}
+
+.bigheader {
+  color: #d91c00;
+}
+
+.salute {
+  color: #f74d1c;
```

Comment:
Kindly follow the guide for color choices. All elements with the orange color needs to have #ec5242 color value.
<img width="432" alt="Screenshot 2023-03-14 at 10 00 12 PM" src="https://user-images.githubusercontent.com/25887253/225111267-f2e85d2b-e2de-4c82-b642-7435a1b66a7b.png">
<img width="432" alt="Screenshot 2023-03-14 at 10 05 06 PM" src="https://user-images.githubusercontent.com/25887253/225111275-f736ee70-5e01-4ec7-890e-a2ff2c5e2bdb.png">